include/rocksdb/filter_policy.h

```diff
@@ -266,12 +280,12 @@ extern const FilterPolicy* NewBloomFilterPolicy(
 // Also consider using optimize_filters_for_memory to save filter
 // memory.
 extern const FilterPolicy* NewRibbonFilterPolicy(
-    double bloom_equivalent_bits_per_key);
+    double bloom_equivalent_bits_per_key, int bloom_before_level = 0);
```

Comment:
I'm wondering if bloom_before_level could be 1 by default. especially for leveled compaction.
And is it okay to change the default behavior?

Reply:
I don't think it's easy to change default only for leveled compaction, and I don't want to mess up single-level universal with the default behavior of NewRibbonFilterPolicy never creating ribbon filters.